data/json/monsters/zed_dormant.json

```diff
@@ -0,0 +1,109 @@
+[
+  {
+    "type": "SPELL",
+    "id": "kill_pseudo_zombie",
+    "name": { "str": "kill pseudo zombie" },
+    "description": "You should not see this.  Kills pseudo zombie to turn it into a corpse for revival.",
+    "valid_targets": [ "self" ],
+    "max_level": 1,
+    "flags": [ "SILENT", "NO_EXPLOSION_SFX", "PERCENTAGE_DAMAGE", "NO_FAIL" ],
+    "base_casting_time": 1,
+    "shape": "blast",
+    "min_damage": 100,
+    "max_damage": 100,
+    "min_aoe": 1,
+    "max_aoe": 1,
+    "message": "",
+    "effect": "attack",
+    "damage_type": "pure"
+  },
+  {
+    "type": "SPELL",
+    "id": "pseudo_dormant_trap_setup",
+    "name": { "str": "dormant corpse setup" },
+    "description": "You should not see this.  Sets up trap for dormant zombies.",
+    "valid_targets": [ "self" ],
+    "max_level": 1,
+    "flags": [ "SILENT", "NO_EXPLOSION_SFX" ],
+    "base_casting_time": 1,
+    "shape": "blast",
+    "min_aoe": 1,
+    "max_aoe": 1,
+    "message": "",
+    "effect": "add_trap",
+    "effect_str": "tr_dormant_corpse",
+    "extra_effects": [ { "id": "kill_pseudo_zombie", "hit_self": true } ]
+  },
+  {
+    "id": "mon_pseudo_dormant_zombie",
+    "type": "MONSTER",
+    "name": { "str": "zombie" },
+    "description": "Fake zombie used for spawning dormant zombies.  If you see this, open an issue on github.",
+    "copy-from": "mon_zombie",
```

Comment:
```suggestion
    "copy-from": "mon_zombie",
    "looks_like": "corpse_generic_human",
```

Reply:
Doesn't work. The corpse is a different item than the zombie itself. Also, the corpse sprite we want is corpse_mon_zombie, not corpse_generic_human, at least for Ultica.

Reply:
As discussed on discord, solving this requires a tileset server update. Out of scope for this PR.